packages/utils/src/config/to-absolute-path.ts

```diff
@@ -0,0 +1,80 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { UserConfig, HintsConfigObject } from '../types/config';
+
+export const toAbsolutePaths = (config: UserConfig | null, configRoot: string): UserConfig | null => {
```

Comment:
Given the method is `toAbsolutePaths`, let's call this file `to-absolute-paths.ts` to include the "s" as well for consistency.